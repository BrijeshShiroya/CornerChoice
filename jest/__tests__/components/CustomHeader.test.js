import 'react-native';
import React from 'react';
import CustomHeader from '../../../app/components/CustomHeader';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('CustomHeader renders correctly', () => {
  const customHeaderComponent = renderer.create(<CustomHeader />).toJSON();
  expect(customHeaderComponent).toMatchSnapshot();
});
