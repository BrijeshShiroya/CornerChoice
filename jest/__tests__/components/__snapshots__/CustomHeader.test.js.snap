// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomHeader renders correctly 1`] = `
<RNCSafeAreaView
  forceInset={
    Object {
      "bottom": "never",
    }
  }
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "rgb(0, 0, 0)",
        "flexDirection": "row",
        "height": 98.57142857142857,
        "justifyContent": "space-between",
        "paddingHorizontal": 24,
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "AvenirNext-Bold",
          "fontSize": 29.57142857142857,
          "textAlign": "center",
        },
        undefined,
      ]
    }
  >
    Header
  </Text>
</RNCSafeAreaView>
`;
